### PR TITLE
Upgrade BOM Stock Report base with Qty to Produce

### PR DESCRIPTION
**Pull request by kickapoo**
This a minor upgrade of BOM Stock Report with the addition of Qty to Produce (Similar to the report in source bom_stock_calculated)

Qty to Produce defualts to 1 and Report behaves as it used. With the other selection there is auto calculation for the other fields.

Also, some coloring added to Item Code for easy reference if BOM line has Enought Part to Produce.

### CODE REVIEW

File: erpnext/manufacturing/report/bom_stock_report/bom_stock_report.js [hunk 1]
```diff
@@ -16,6 +16,22 @@ frappe.query_reports["BOM Stock Report"] = {
 			"fieldname": "show_exploded_view",
 			"label": __("Show exploded view"),
 			"fieldtype": "Check"
+		}, {
+					"fieldname": "qty_to_produce",
```
Comment (rmehta) [inline erpnext/manufacturing/report/bom_stock_report/bom_stock_report.js]: please fix indenting

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/manufacturing/report/bom_stock_report/bom_stock_report.js (hunk 1 above)
Comment (kickapoo) [reply erpnext/manufacturing/report/bom_stock_report/bom_stock_report.js]: https://github.com/frappe/erpnext/pull/15697/commits/1547acdec8ced8a680affe12fe15ceb312844d02

**Comment by nabinhait** [comment]
@kickapoo Can you please send the PR to staging-fixes branch?

**Comment by nabinhait** [comment]
closing this

**Comment by kickapoo** [comment]
@nabinhait why this closed?

**Pull request closed without merging**